clusterous/cluster.py

```diff
@@ -254,12 +255,9 @@ def validate_config(fields):
             message = 'Error in S3 bucket name "{0}": {1}'.format(fields['clusterous_s3_bucket'], s3_msg)
             return False, message
 
-        if not vpc_re.search(fields['vpc_id']):
+        if fields.get('vpc_id') and not vpc_re.search(fields['vpc_id']):
```

Comment:
fields.get('vpc_id') is redundant because we have already ensured that vpc_id is in fields. Should be consistent with all the other fields.